.env-dist

```
@@ -149,6 +149,9 @@ STATS_TOKEN=
 # Support article link
 NEXT_PUBLIC_MONITOR_SUPPORT_URL=https://support.mozilla.org/kb/firefox-monitor
 
+# Support How it Works page
+HOW_IT_WORKS_SUMO_URL=https://support.mozilla.org/en-US/kb/how-does-monitor-plus-work
```

nit: remove the /en-US/ localized URL and let SuMo negotiate language.
```suggestion
HOW_IT_WORKS_SUMO_URL=https://support.mozilla.org/kb/how-does-monitor-plus-work
```

---

Also, does this need a `NEXT_PUBLIC_` prefix, a'la `NEXT_PUBLIC_MONITOR_SUPPORT_URL` on L150 above?